docs/Hosts/Spawn-Hosts.md

```diff
@@ -50,14 +50,34 @@ Clicking it will pre-populate the spawn host page with a request to spawn a host
 
 ![spawn_host_modal.png](../images/spawn_host_modal.png)
 
-Fetching artifacts can also be performed manually; see [fetch](../CLI#fetch) in the Evergreen command line tool documentation.
+The spawn host will not fetch the binaries and artifacts associated with the task on start-up because it lacks the necessary permissions. After you SSH into the host, you will need to run the following command to fetch them (which will prompt you to authenticate):
+
+```sh
+evergreen host fetch
+```
+
+Note: You will need to run this command for each spawn host you create that fetches the artifacts of a task.
+
+If you do not want the pre-made fetching options, you can use the Evergreen CLI command [fetch](../CLI#fetch).
 
 Artifacts are placed in /data/mci. Note that you will likely be able to ssh into the host before the artifacts are finished fetching.
 
 If your project has [a project setup script defined at the admin level](../Project-Configuration/Project-And-Distro-Settings.md#spawn-host-script-path), you can also check "Use project-specific setup script defined at ..." before creating the spawn host to run that script when the host starts up. You can check if there are errors fetching artifacts or running this script on the host page: `https://spruce.mongodb.com/host/<host_id>`.
 
 EC2 spawn hosts can be stopped/started and modified from the Spawn Host page, or via the command line, which is documented in [Basic Host Usage](../CLI#basic-host-usage) in the Evergreen command line tool documentation.
 
+## Evergreen CLI
+
+When using the Evergreen CLI on a spawn host, you will be prompted to authenticate by going to a URL and entering a code. If the link does not appear in the terminal, you may have to set `oauth.do_not_use_browser` to true in your Evergreen CLI config file (usually located at `~/.evergreen.yml`). Example:
+
+```yaml
+# ~/.evergreen.yml
+# ...
+oauth:
+  do_not_use_browser: true
```

Comment:
nit: let's make sure we have the work to add this automatically tracked.

Reply:
Wdym by this 👁️ 👁️ ?

Reply:
Users should not need to manually specify this, when we create the ~/.evergreen.yml file for spawn hosts we should add this automatically. I believe we already have a ticket for this (or already doing this somewhere) but I just wanted to double check.

Reply:
Ah, this PR already adds it when it populates the spawnhost config. This documentation is for old spawn hosts, since we can't detect spawn host vs manual user.